src/Features/Onboarding.php

```diff
@@ -105,6 +109,15 @@ private function add_actions() {
 		// Rest API hooks need to run before is_admin() checks.
 		add_action( 'woocommerce_theme_installed', array( $this, 'delete_themes_transient' ) );
 		add_action( 'after_switch_theme', array( $this, 'delete_themes_transient' ) );
+		add_action(
```

Comment:
Minor but when specifying a different priority for a hook to operate, it is nice to add a comment explaining the reasoning/background for that decision. Just helps with future us debugging reasoning for certain actions happening before/after others.

Reply:
The default priority is 10 which is what is being used here. The only reason the priority is set here is to change the number of arguments to 2, otherwise the hook doesn't work properly.

Reply:
Ah that is right, don't mind me.